src/Mayaqua/CMakeLists.txt

```diff
@@ -63,6 +63,8 @@ if(WIN32)
       "setupapi.lib"
       "winmm.lib"
       "WtsApi32.Lib"
+      "ws2_32.lib"
+      "crypt32.lib"
```

Comment:
The libraries are actually in alphabetical order, I would keep it.

Reply:
OK